docs/ops/activation/Swish_4.md

```diff
@@ -0,0 +1,54 @@
+## Swish <a name="Swish"></a>
+
+**Versioned name**: *Swish-4*
+
+**Category**: *Activation*
+
+**Short description**: Swish takes one input tensor and produces output tensor where the swish function is applied to the tensor elementwise.
+
+**Detailed description**: [Reference.](https://arxiv.org/pdf/1710.05941.pdf)    
+For each element from the input tensor calculates corresponding
+element in the output tensor with the following formula: `x * sigmoid(beta * x)`
```

Comment:
Please, be consistent with the formula style. For example, in the review for SoftPlus you used different approach.

Reply:
Formula updated.